cmd/snap/cmd_debug_timings.go

```diff
@@ -152,6 +165,9 @@ func (x *cmdChangeTimings) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	if x.StartupTag != "" && (x.Positional.ID != "" || x.EnsureTag != "") {
```

Comment:
I'd say, either have one single "these three options are mutually exclusive" error, or have one for each invalid combination. Otherwise it's a bit weird.

While I'm at it I'll point out that the error when you specify no options is a bit misleading / frustrating, as it comes exclusively from `GetChangeID`,

    ~$ snap debug timings 
    error: please provide change ID or type with --last=<type>

(no mention of `--statup`/`--ensure`)

Reply:
Fair point, that's improved now.